Flutter/lib/SettleScreen.dart

```diff
@@ -45,45 +41,96 @@ class _SettleScreenState extends State<SettleScreen> {
               onBack: (index) {
                 setState(() {});
               },
-              onEnd: () { // TODO: go to result screen
+              onEnd: () {
+                // TODO: go to result screen
                 print('end');
                 settle.userFinished();
-                Navigator.push( // Go to results screen
-                  context,
-                  MaterialPageRoute(builder: (context) => ResultScreen(settle))
-                );
+                Navigator.push(
+                    // Go to results screen
+                    context,
+                    MaterialPageRoute(
+                        builder: (context) => ResultScreen(settle)));
               },
             ),
             SizedBox(height: 70),
             Row(
               mainAxisAlignment: MainAxisAlignment.spaceEvenly,
               children: <Widget>[
-                FloatingActionButton(
-                  heroTag: Text("nope"),
-                  // this will be forward and it should indicate user didn't
-                  // like it. For demo, it's set to back
-                  onPressed: () {
-                    _controller.back();
-                  },
-                  backgroundColor: Colors.white,
-                  child: Icon(Icons.close, color: Colors.red),
+                NeumorphicButton(
+                  onPressed: _controller.back,
+                  style: NeumorphicStyle(
+                    depth: 2.7,
+                    intensity: 0.35,
+                    surfaceIntensity: 0.5,
+                    lightSource: LightSource.topLeft,
+                    shadowLightColor: widget.themeChange.darkTheme
+                        ? Colors.white
+                        : Colors.black,
+                    shadowDarkColor: Colors.grey[900],
+                    color: widget.themeChange.darkTheme
+                        ? Color(0xff1E1E1E)
+                        : Colors.grey[200],
```

Comment:
These seem like things that AppTheme should be handling

Reply:
I feel this is an exception. There's no match between either ternary operators. If I make a method or constant in AppTheme, it'll be used exclusively for this instance. If you want you can change them however you like